checks.xml

```diff
@@ -264,25 +259,22 @@
             <property name="allowSamelineMultipleAnnotations" value="true"/>
         </module>
         <module name="NonEmptyAtclauseDescription"/>
-        <module name="JavadocTagContinuationIndentation"/>
+        <module name="JavadocTagContinuationIndentation">
+            <property name="severity" value="ignore"/>
+        </module>
         <module name="SummaryJavadoc">
-            <property name="forbiddenSummaryFragments"
```

Comment:
Same here. Why?

Reply:
see above
The check is useless and generated too many warnings